DQMOffline/Trigger/plugins/MssmHbbMonitor.cc

```diff
@@ -0,0 +1,203 @@
+#include <math.h>
+
+#include "FWCore/Framework/interface/EventSetup.h"
+
+#include "DataFormats/Common/interface/Handle.h"
+
+
+#include "DQMOffline/Trigger/plugins/MssmHbbMonitor.h"
+
+#include "TLorentzVector.h"
+
+
+// -----------------------------
+//  constructors and destructor
+// -----------------------------
+
+MssmHbbMonitor::MssmHbbMonitor( const edm::ParameterSet& iConfig )
+{
+  folderName_             = iConfig.getParameter<std::string>("dirname");
+  processname_            = iConfig.getParameter<std::string>("processname");
+  pathname_               = iConfig.getParameter<std::string>("pathname");
+  jetPtmin_               = iConfig.getParameter<double>("jetPtMin");
+  jetEtamax_              = iConfig.getParameter<double>("jetEtaMax");
+  muonPtmin_              = iConfig.getParameter<double>("muonPtMin");
+  muonEtamax_             = iConfig.getParameter<double>("muonEtaMax");
+  jetMuonDRmax_           = iConfig.getParameter<double>("jetMuonDRMax");
+  triggerResultsToken_    = consumes <edm::TriggerResults>   (iConfig.getParameter<edm::InputTag>("triggerResults"));
+  muonsToken_             = consumes<reco::MuonCollection>   (iConfig.getParameter<edm::InputTag>("muons"));
+  offlineBtagToken_       = consumes<reco::JetTagCollection> (iConfig.getParameter<edm::InputTag>("offlineBtag"));
+
+  jetPtbins_              = getHistoPSet(iConfig.getParameter<edm::ParameterSet>("histoPSet").getParameter<edm::ParameterSet>("jetPt"));
```

Comment:
I would really suggest to make use of variable binning as much as you can
we have issue w/ memory ....